### DISCUSSION TITLE
Problem with changed themes variables

### DISCUSSION THREAD

**Discussion by PhoenixReal** [NvChad/NvChad#2836] [Q&A] [answered]
I have this issue with my chadrc.lua. When I try to set nvdash startup enable it
![Screenshot_2](https://github.com/NvChad/NvChad/assets/167446860/c0a0fc2c-bffe-4e07-b16a-e56cb81f468d)

**Comment by siduck** [comment, marked as answer]
its normal, please dont keep tables empty if you want the errors to go,  only add options in your chadrc which you changed as chadrc table already merges with the nvconfig one.. So u dont have to repeat the default stuff again
